Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -429,6 +434,10 @@ struct PerWebProcessState {
     WebCore::FixedContainerEdges _fixedContainerEdges;
 
     RetainPtr<WKScrollGeometry> _currentScrollGeometry;
+
+#if ENABLE(PDF_PAGE_NUMBER_INDICATOR)
+    WTF::HashMap<WebKit::PDFPluginIdentifier, RetainPtr<WKPDFPageNumberIndicator>> _pdfPageNumberIndicators;
```

Comment:
Why would there be more than one? If may one OK to parent the main frame one directly under the web view, but you can't do that for subframes because the indicator won't scroll with the frame.

Reply:
Indeed. I originally wrote it this way hoping it would simplify the embedded PDF case, but it won't.

I've changed this to just be a `std::pair<Markable<PluginIdentifier>, RetainPtr<WKPDFPageNumberIndicator>>` instead.